recipe/bld.bat

```diff
@@ -0,0 +1,38 @@
+@ECHO ON
+
+@REM I cannot for the life of me figure out how Cygwin/MSYS2 figures out its
```

Comment:
Cygwin/MSYS2 locate themselves entirely from the location of the {cygwin-1.0,msys-2.0}.dll